src/file/drawing/inline/graphic/graphic-data/pic/shape-properties/no-fill.ts

```diff
@@ -1,7 +0,0 @@
-import { XmlComponent } from "file/xml-components";
```

Comment:
I deleted it because I think it was a duplicate of a class that was already created and it didn't seem to be used anywhere.  I found that it was a duplicate b/c the TS compiler was yelling at me about conflicting exports.  If it's not a duplicate, prob should be added back in?

Reply:
Yes, you're right, its a duplicate, there is another one else where